prereceivecli/prereceivecli.py

```diff
@@ -77,6 +89,37 @@ def get_arguments():
     # https://docs.python.org/3/library/argparse.html
     parser = argparse.ArgumentParser(description=('A cli that implements a git server side pre-receive hook that gets '
                                                   'driven from dynamodb and reports to slack offending pushes.'))
+    mutex_group1 = parser.add_mutually_exclusive_group(required=True)
```

Comment:
Please rename these to something that semantically makes sense like ```excluding_key_arn``` and so on for the rest.